server/routes/user.js

```diff
@@ -20,6 +21,13 @@ router.post("/review", Auth, async (req, res) => {
       title: req.body.title,
       message: JSON.stringify(req.body.content.text),
     };
+    await createNotification({
+      // recipient: userId,
+      recipient: "5f3c45372421c5628414a45f",
+      thread: "Thread",
+      text: "Text",
+      author: "Author",
+    });
```

Comment:
this should only happen when a reviewer is found, which is in the find reviewer queue file

Reply:
Yeah, that's obvious. I've put it here to test if it works correctly, bc reviewer matching works inadequately (the same user gets request). Also I think we will have another notifications too - like 'u have new message', 'your code was reviewed' etc..

Reply:
ok it will better if we can see it working with the queue in my opinion. In terms of reviewer matching, the BE works fine except for the userLanguageLevel (we need to think on how we can compare it - using numbers for each level e.g. userLanguageLevel: 1 if it is a Beginner). 

The reason you see the same user having the accept and reject buttons currently is because of the FE logic, which will be fixed moving forward.

My suggestion is to try it with the queue, I can help you with this if needed.